code/src/constants/common.ts

```diff
@@ -1,3 +1,10 @@
 export const LOCAL_STORAGE_KEYS = {
   userDetails: "spa-user-details"
-};
+} as const;
+
+// @TODO: rewrite with translations
+export const ERROR_MESSAGES_BY_STATUS = {
+  400: "Invalid input data. Please check it and try again",
+  401: "User is not authorized to perform this action",
+  409: "Already exists"
```

Comment:
think about translates for other users ? 
It should be translates for your code error value

Reply:
I added a todo for it. The new issue should be created, because it involves refactoring `AppSnackbar` component, which is not tightly related to my task

Reply:
> I added a todo for it. The new issue should be created, because it involves refactoring `AppSnackbar` component, which is not tightly related to my task

Keep in mind. We resolve issues with translates currently for the PR and not create a new issue for that.
Is it so problem to resolve it here ?